starter-project-web-vue2/components/aymen/BlogList.vue

```diff
@@ -1,6 +1,16 @@
 <template>
   <v-container>
-    <h2>Latest Blogs</h2>
+    <v-toolbar flat app>
```

Comment:
also, vuetify is complaining `flat` class is depreciated in the current version, use `text` instead